test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageSearchServiceTests.cs

```diff
@@ -63,7 +63,7 @@ public NuGetPackageSearchServiceTests(GlobalServiceProvider globalServiceProvide
             _sourceRepository = StaticHttpHandler.CreateSource(testFeedUrl, Repository.Provider.GetCoreV3(), responses);
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/NuGet/Home/issues/10450")]
```

Comment:
```suggestion
        [Fact(Skip = "https://github.com/NuGet/Home/issues/10451")]
```

Reply:
@zivkan 

Ok. I have taken your recommendation. Can you approve now?